components/Tabs2/src/private/MDCTabBarViewItemView.h

```diff
@@ -17,4 +17,10 @@
 /** A basic view that displays a title and image for a tab bar item within MDCTabBarView. */
 @interface MDCTabBarViewItemView : UIView
 
+/** Title for the tab. Defaults to the empty string. */
+@property(nonatomic, copy, nonnull) NSString *title;
```

Comment:
I think this should be nullable. The [Material Guidelines article shows Tabs without any title text](https://material.io/design/components/tabs.html#anatomy) as a potential option..

<img width="796" alt="Screen Shot 2019-06-20 at 10 10 57 PM" src="https://user-images.githubusercontent.com/1753199/59892726-4d2b8f80-93a8-11e9-8f11-7bc98f2382e1.png">

Reply:
Done.